api/bonsai_api/redis/models.py

```diff
@@ -0,0 +1,30 @@
+"""Functions for handling redis jobs."""
+
+from enum import Enum
+
+from ..models.base import RWModel
+
+
+class SubmittedJob(RWModel):  # pylint: disable=too-few-public-methods
```

Comment:
Turn off the `too-few-public-methods` in a config?

Reply:
Can it be done selectively for only specific files or functions?

Reply:
I think so, adding the rule to the top of the file (only read about it, I haven't tested it)